Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python
+
+RUN mkdir /tmp/dcos-cli
+RUN apt-get update && apt-get install -y wget curl python-pip less tmux
+
+RUN pip install virtualenv httpie
+
+ENV DCOS_CONFIG=/root/.dcos/dcos.toml
+
+ADD https://s3.amazonaws.com/downloads.mesosphere.io/dcos-cli/install.sh /tmp/
+RUN /bin/bash /tmp/install.sh /tmp/dcos-cli http://localhost || true
+
+RUN wget -qO- https://downloads.mesosphere.io/java/jre-8u51-linux-x64.tar.gz | tar xvz -C /opt/
```

Comment:
We'll probably need to change to that. It really shouldn't have to be there, but it's needed by the kafka cli.